lighthouse-core/test/gather/fake-driver.js

```diff
@@ -5,13 +5,16 @@
  */
 'use strict';
 
-function makeFakeDriver({protocolGetVersionResponse}) {
+function makeFakeDriver({protocolGetVersionResponse, driverGetPageUAResponse}) {
   let scrollPosition = {x: 0, y: 0};
 
   return {
     getBrowserVersion() {
       return Promise.resolve(Object.assign({}, protocolGetVersionResponse, {milestone: 71}));
     },
+    getPageUserAgent() {
+      return Promise.resolve(driverGetPageUAResponse);
```

Comment:
it's just for the one method basically, instead of modifying the signature should we just override `getPageUserAgent` for the fake driver there with `jest.fn().mockResolvedValue('User Agent')`?

the protocol response was a little more involved and made sense to be shared IMO :)